packages/bpk-tokens/src/android/base/typography.json

```diff
@@ -63,36 +63,6 @@
       "type": "number",
       "category": "typesettings"
     },
-    "LETTER_SPACING_XS": {
```

Comment:
Remember and add a changelog entry! (calling this out as this is a breaking change)